ajax/libs/swiftclick/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "swiftclick",
+  "description": "Eliminates the 300ms click event delay on touch devices that support orientation change",
+  "author": {
+    "name": "Ivan Hayes",
+    "url": "http://www.ivanhayes.com"
+  },
+  "keywords": [
+    "swiftclick",
+    "fastclick",
+    "mobile",
+    "tap",
+    "touch",
+    "click",
+    "delay"
+  ],
+  "filename": "swiftclick.min.js",
+  "version": "1.4.1",
```

Comment:
Since the assets didn't be committed in this PR so please remove `version` to make it been ignored.